cmd/backup-manager/app/util/util.go

```diff
@@ -298,6 +298,10 @@ func GetOptions(provider v1alpha1.StorageProvider) []string {
 	switch st {
 	case v1alpha1.BackupStorageTypeS3:
 		return provider.S3.Options
+	case v1alpha1.BackupStorageTypeGcs:
+		return provider.Gcs.Options
+	case v1alpha1.BackupStorageTypeLocal:
+		return provider.Local.Options
```

Comment:
Currently, the `options` is used for rclone, so no need to update for local storage as it does not use rclone.

Reply:
yes, options of local provider has been removed.